go.mod

```diff
@@ -3,38 +3,37 @@ module sigs.k8s.io/cluster-api-provider-azure
 go 1.13
 
 require (
-	github.com/Azure/azure-sdk-for-go v46.4.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.7
-	github.com/Azure/go-autorest/autorest/azure/auth v0.5.2
+	github.com/Azure/azure-sdk-for-go v48.2.0+incompatible
+	github.com/Azure/go-autorest/autorest v0.11.11
+	github.com/Azure/go-autorest/autorest/azure/auth v0.5.3
 	github.com/Azure/go-autorest/autorest/to v0.4.0
 	github.com/Azure/go-autorest/autorest/validation v0.3.0 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0
 	github.com/blang/semver v3.5.1+incompatible
 	github.com/go-logr/logr v0.1.0
 	github.com/golang/mock v1.4.4
-	github.com/google/go-cmp v0.5.2
+	github.com/google/go-cmp v0.5.3
 	github.com/google/gofuzz v1.2.0
 	github.com/google/uuid v1.1.2
-	github.com/hashicorp/go-retryablehttp v0.6.7
-	github.com/onsi/ginkgo v1.14.1
-	github.com/onsi/gomega v1.10.2
+	github.com/hashicorp/go-retryablehttp v0.6.8
+	github.com/onsi/ginkgo v1.14.2
+	github.com/onsi/gomega v1.10.3
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/pflag v1.0.5
 	go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp v0.13.0
 	go.opentelemetry.io/otel v0.13.0
 	go.opentelemetry.io/otel/exporters/metric/prometheus v0.13.0
 	go.opentelemetry.io/otel/exporters/trace/jaeger v0.13.0
 	go.opentelemetry.io/otel/sdk v0.13.0
-	golang.org/x/crypto v0.0.0-20201002170205-7f63de1d35b0
-	golang.org/x/net v0.0.0-20201002202402-0a1ea396d57c // indirect
-	k8s.io/api v0.17.12
-	k8s.io/apimachinery v0.17.12
-	k8s.io/client-go v0.17.12
-	k8s.io/component-base v0.17.12
+	golang.org/x/crypto v0.0.0-20201117144127-c1f2f97bffc9
+	k8s.io/api v0.17.14
```

Comment:
should we upgrade this to 0.18.x ? or we need to stay on 1.17 for any reason that i don't know :)

Reply:
1.17 is fine for now, let's stick to the same minor version as CAPI 0.3 https://github.com/kubernetes-sigs/cluster-api/blob/release-0.3/go.mod#L35

Reply:
Perhaps, we should stay on 1.17 until we jump to 1.19 following CAPI (https://github.com/kubernetes-sigs/cluster-api/blob/e6b0039b4aa1021b0a9afec8a527d969c20acabb/go.mod#L32)

Reply:
thanks for the clarification @devigned @CecileRobertMichon